manifests/cf-manifest/operations.d/720-rds-broker-postgres12-plans.yml

```diff
@@ -290,7 +350,7 @@
         - "AWS RDS"
     rds_properties:
       <<: *default_postgres_rds_properties # yamllint disable-line
-      <<: *large_plan_rds_properties # yamllint disable-line
+      <<: *old_large_plan_rds_properties # yamllint disable-line
```

Comment:
Should we change this to be `deprecated_large_plan_rds_properties` to make it linguistically consistent